orchestrator/publisher/application/services/organization_service.py

```diff
@@ -8,6 +8,7 @@
     SLACK_CHANNEL_FOR_APPROVAL_TEAM, NOTIFICATION_ARN, ORG_APPROVERS_DLIST
 from orchestrator.constant import VerificationType, OrganizationType
 from orchestrator.publisher.mail_templates import get_org_approval_mail
+from orchestrator.publisher.mail_templates import get_notification_mail_template_for_service_provider_when_org_is_submitted_for_onboarding
```

Comment:
get_mail_template_to_user_for_org_onboarding() will be smaller and good name for method